src/views/Languages/index.jsx

```diff
@@ -0,0 +1,142 @@
+  BUGZILLA_SEARCH_OPTIONS,
+} from '../../utils/constants';
+import extractWhiteboardTags from '../../utils/extractWhiteboardTags';
+
+@withApollo
+@hot(module)
+@graphql(bugsQuery, {
+  skip: ({
+    match: {
+      params: { language },
+    },
+  }) => !BUGZILLA_LANGUAGES[language],
+  name: 'bugzilla',
+  options: ({
+    match: {
+      params: { language },
+    },
+  }) => ({
+    fetchPolicy: 'network-only',
+    variables: {
+      goodFirst: {
+        ...BUGZILLA_SEARCH_OPTIONS,
+        keywords: [GOOD_FIRST_BUG],
+        whiteboards: `lang=${BUGZILLA_LANGUAGES[language]}`,
+      },
+      mentored: {
+        ...BUGZILLA_SEARCH_OPTIONS,
+        ...MENTORED_BUG,
+        whiteboards: `lang=${BUGZILLA_LANGUAGES[language]}`,
+      },
+      paging: {
+        ...BUGZILLA_PAGING_OPTIONS,
+      },
+    },
+    context: {
+      client: 'bugzilla',
+    },
+  }),
+})
+export default class Languages extends Component {
+  state = {
+    error: null,
+  };
+
+  handleBugInfoClick = memoizeWith(
+    id => id,
+    async id => {
+      try {
+        const {
+          data: { comments },
+        } = await this.props.client.query({
+          query: commentsQuery,
+          variables: { id },
+          context: { client: 'bugzilla' },
+        });
+
+        return comments[0].text;
+      } catch (error) {
+        this.setState({ error });
+      }
+    }
+  );
+
+  render() {
+    const {
+      match: {
+        params: { language },
+      },
+      bugzilla: bugzillaData,
+    } = this.props;
+    const { error } = this.state;
+    const goodFirstBugs =
+      (bugzillaData &&
+        bugzillaData.goodFirst &&
+        uniqBy(
+          bugzillaData.goodFirst.edges.map(edge => edge.node).map(bug => ({
+            assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
+            project: bug.component,
+            tags: [
+              ...(bug.keywords || []),
+              ...extractWhiteboardTags(bug.whiteboard),
+            ],
+            summary: bug.summary,
+            lastUpdated: bug.lastChanged,
+            id: bug.id,
+            url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+          })),
+          'summary'
+        )) ||
+      [];
+    const mentoredBugs =
+      (bugzillaData &&
+        bugzillaData.mentored &&
+        uniqBy(
+          bugzillaData.mentored.edges.map(edge => edge.node).map(bug => ({
+            assignee: bug.status === 'ASSIGNED' ? bug.assignedTo.name : '-',
+            project: bug.component,
+            tags: [
+              ...(bug.keywords || []),
+              ...extractWhiteboardTags(bug.whiteboard),
+            ],
+            summary: bug.summary,
+            lastUpdated: bug.lastChanged,
+            id: bug.id,
+            url: `https://bugzilla.mozilla.org/show_bug.cgi?id=${bug.id}`,
+          })),
+          'summary'
+        )) ||
+      [];
+
+    return (
+      <Dashboard activeItem={language} withSidebar>
```

Comment:
No need provide the `activeItem` prop. I believe you can infer this from the last parameter of the URL.